src/components/signup-view/signup-view.jsx

```diff
@@ -16,8 +19,7 @@ export const SignupView = () => {
       Birthday: birthday,
     };
```

Comment:
Your API: https://github.com/allmyprogramming/movie_api/blob/14dd9c075086c77494496461bf045f4ee0a5d969/index.js#L142C1-L142C76  expects the keys to be lower case, so this would also fail when you're testing user signup in your application.